packages/jaeger-ui/src/components/SearchTracePage/index.test.js

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
I don't think the changes to this file are still necessary.

Reply:
oh I see, it's because a child of this component changed and mount makes that relevant... I'll see if I can get rid of mount.

Reply:
I see, without this the `withRouter` in `SearchResults` returns something that enzyme dislikes.